### PR TITLE
Update to support Sylius 1.9 and Symfony 5.2

### PR DESCRIPTION
**Pull request by pptasinski-bitbag**

### CODE REVIEW

File: tests/Application/config/bundles.php [hunk 1]
```diff
@@ -40,7 +39,7 @@
     Liip\ImagineBundle\LiipImagineBundle::class => ['all' => true],
     Payum\Bundle\PayumBundle\PayumBundle::class => ['all' => true],
     Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle::class => ['all' => true],
-    WhiteOctober\PagerfantaBundle\WhiteOctoberPagerfantaBundle::class => ['all' => true],
+    BabDev\PagerfantaBundle\BabDevPagerfantaBundle::class => ['all' => true],
```
Comment (damonsson) [inline tests/Application/config/bundles.php:42]: Why changed vendor?

File: src/Entity/Wishlist.php [hunk 2]
```diff
@@ -14,23 +14,23 @@
...
     /** @var Collection|WishlistProductInterface[] */
     protected $wishlistProducts;
 
-    /** @var ShopUserInterface|null */
-    protected $shopUser;
+    protected ?ShopUserInterface $shopUser;
 
-    /** @var TokenInterface|null */
+    /** @var TokenInterface|null
+     * @ORM\Column(type="string") */
```
[the remark this answers is not in the document]
Comment (pptasinski-bitbag) [reply src/Entity/Wishlist.php:33]: Good catch - accidentaly shortcut pressed :D

File: spec/EventListener/ClearCookieWishlistItemsListenerSpec.php [hunk 3]
```diff
@@ -6,10 +6,12 @@
 
 use BitBag\SyliusWishlistPlugin\EventListener\ClearCookieWishlistItemsListener;
 use PhpSpec\ObjectBehavior;
-use Sylius\Component\Core\Model\AdminUserInterface;
-use Sylius\Component\Core\Model\ShopUserInterface;
+use Sylius\Component\Core\Model\AdminUser;
```
[the remark this answers is not in the document]
Comment (pptasinski-bitbag) [reply spec/EventListener/ClearCookieWishlistItemsListenerSpec.php:9]: InteractiveLoginEvent is final so mocking is not possible

File: tests/Application/config/bundles.php (hunk 1 above)
Comment (pptasinski-bitbag) [reply tests/Application/config/bundles.php:42]: Required by upgrade to 1.9

File: spec/EventListener/MergeUserWishlistItemsListenerSpec.php [hunk 4]
```diff
@@ -12,11 +12,12 @@
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\ORM\EntityManagerInterface;
 use PhpSpec\ObjectBehavior;
-use Sylius\Component\Core\Model\AdminUserInterface;
-use Sylius\Component\Core\Model\ShopUserInterface;
+use Sylius\Component\Core\Model\AdminUser;
```
[the remark this answers is not in the document]
Comment (pptasinski-bitbag) [reply spec/EventListener/MergeUserWishlistItemsListenerSpec.php:15]: InteractiveLoginEvent is final so mocking is not possible

**Review by damonsson — APPROVED** [review]

**Pull request merged**